README.md

```diff
@@ -34,7 +34,7 @@ libpod is currently in active development.
 | Command                                                  | Description                                                               | Demo|
 | :------------------------------------------------------- | :------------------------------------------------------------------------ | :----|
 | [podman(1)](/docs/podman.1.md)                           | Simple management tool for pods and images                                ||
-| [podman-attach(1)](/docs/podman-attach.1.md)             | Attach to a running container                                             ||
+| [podman-attach(1)](/docs/podman-attach.1.md)             | Attach to a running container                                             |[![...](/docs/play.png)](https://asciinema.org/a/XDlocUrHVETFECg4zlO9nBbLf)|
```

Comment:
This video has about 64 lines in it which causes it to scroll.  Can you reshoot it to make it 24 lines?

Reply:
Only if you guys really demand it.